### DISCUSSION TITLE
Notifiche valori variabili con PushBullet

### DISCUSSION THREAD

**Discussion by moresco70** [indomus/forum#1852] [HUB: Home Assistant]
Ciao a tutti,
ho configurato con successo il servizio di pushbullet ma non riesco a farmi inviare le notifiche dove, al posto di un messaggio preconfigurato, possa inviare il valore posseduto da una variabile di un determinato sensore.
Ho provato a leggere anche la doc sulle API non ho trovato nulla. 
Qualcuno è riuscito nell'intento?
Grazie

**Comment by snakuzzo** [comment]
Mai configurata/utilizzata questa integrazione, ma immagino tu possa utilizzare i [template](https://www.home-assistant.io/docs/configuration/templating/) come per qualsiasi altra cosa.
